packages/itwinui-react/src/core/Tabs/Tabs.tsx

```diff
@@ -526,85 +511,360 @@ export const Tabs = (props: TabsProps) => {
-            React.cloneElement(label as JSX.Element, {
-              active: index === currentActiveIndex,
-              'aria-selected': index === currentActiveIndex,
-              tabIndex: index === currentActiveIndex ? 0 : -1,
-              onClick: (args: unknown) => {
-                onClick();
-                label.props.onClick?.(args);
-              },
-            })
-          )}
-        </li>
-      );
-    },
-    [currentActiveIndex, onTabClick],
+  return (
+    <Box
+      as='ul'
+      className={cx(
+        'iui-tabs',
+        `iui-${type}`,
+        {
+          'iui-green': color === 'green',
+          'iui-animated': type !== 'default' && isClient,
+          'iui-not-animated': type !== 'default' && !isClient,
+          'iui-large': hasSublabel,
+        },
+        className,
+      )}
+      data-iui-overflow={overflowOptions?.useOverflow}
+      data-iui-scroll-placement={scrollingPlacement}
+      role='tablist'
+      onKeyDown={onKeyDown}
+      ref={refs}
+      {...rest}
+    >
+      {items.map((item, index) => {
+        return (
+          <li key={index}>
+            <TabsContext.Provider
+              value={{
+                index,
+                currentActiveIndex,
+                setCurrentActiveIndex,
+                onTabSelected: onTabClick,
+                setFocusedIndex,
+              }}
+              key={index}
+            >
+              {item}
+            </TabsContext.Provider>
+          </li>
+        );
+      })}
+    </Box>
   );
+}) as PolymorphicForwardRefComponent<'ul', TabsTabListOwnProps>;
+TabsTabList.displayName = 'Tabs.TabList';
+
+// ----------------------------------------------------------------------------
+// Tabs.Tab component
+
+type TabsTabOwnProps = {}; // eslint-disable-line @typescript-eslint/ban-types
+
+const TabsTab = React.forwardRef((props, ref) => {
+  const { className, children, ...rest } = props;
 
+  const { index, currentActiveIndex, onTabSelected, setFocusedIndex } =
+    useSafeContext(TabsContext);
+
+  const onClick = () => {
+    if (index !== undefined) {
+      setFocusedIndex && setFocusedIndex(index);
+      onTabSelected && onTabSelected(index);
+    }
+  };
   return (
     <Box
-      className={cx('iui-tabs-wrapper', `iui-${orientation}`, wrapperClassName)}
-      style={stripeProperties}
+      as='button'
+      className={cx(
+        'iui-tab',
+        { 'iui-active': index === currentActiveIndex },
+        className,
+      )}
+      role='tab'
+      tabIndex={index === currentActiveIndex ? 0 : -1}
+      onClick={onClick}
+      aria-selected={index === currentActiveIndex}
+      ref={ref}
+      {...rest}
     >
+      {children}
+    </Box>
+  );
+}) as PolymorphicForwardRefComponent<'button', TabsTabOwnProps>;
+TabsTab.displayName = 'Tabs.Tab';
+
+// ----------------------------------------------------------------------------
+// Tabs.TabIcon component
+
+const TabsTabIcon = polymorphic.span('iui-tab-icon', {
```

Comment:
You should rething Icon, Info, Label and Description components. The current approach is very verbose and a lot of code needed for simple stuff. Maybe we could achieve something like this:
```
<Tab label='Label 1' />
```
```
<Tab>
  <Tab.Icon>☆</Tab.Icon>
  <Tab.Label>Label 2</Tab.Label>
  <Tab.Description>Description</Tab.Description>
</Tab>
```

Reply:
I agree that creating basic tabs with no description and no icon should use the first example, but I think `Tab.Label` and `Tab.Description` need to be wrapped within a `div` element with the class name `iui-tab-label` so that the label and the description both get the `display: block` property.

Reply:
I've added a `label` prop to the `Tabs.Tab` subcomponent so simple tabs can be less verbose.